### ISSUE TITLE
mask_to_tensor : function explanation

### ISSUE THREAD

**Issue by xiching** [albu/albumentations#85]
Hi,
Very interesting and thorough package for playing with images. 
I am curious to know what is the purpose of the function `mask_to_tensor` in `.torch.functional`? More specifically, what does it mean to create a mask, like:
```
for c in range(mask.shape[2]):
    long_mask[mask[..., c] > 0] = c
```
  and where/why would I use such a weird looking mask? 

Please bear in mind that I am not asking how this code work but that _why_ would someone ever need to create such a mask? Application of the mask. Few examples would be enlightening, thanks!

https://github.com/albu/albumentations/blob/master/albumentations/torch/functional.py#L15

**Comment by albu** [comment]
Hi, @xiching resulting array is a target for pytorch's CrossEntropyLoss - https://pytorch.org/docs/stable/nn.html#torch.nn.CrossEntropyLoss . 

Originally this mask was encoded as multi-channel array, but now I thinks this code is not useful most of the time because masks are usually already encoded in this manner (single-channel)
